dipy/align/api.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import nibabel as nib
+
+from imwarp import SymmetricDiffeomorphicRegistration
+from imwarp import DiffeomorphicMap
+from metrics import CCMetric
+import vector_fields as vfu
+
+
+def syn_registration(moving, static, metric=CCMetric(3), 
```

Comment:
Here you have it.
Here is a dummy example. I am assuming that someone needs to do a very simple processing. For example combine fa and t1 from the same subjects, do some processing with them and then create some output in an output_dir. 
Here is the very thin call from the CLI
https://github.com/Garyfallidis/dipy/blob/interfaces/bin/dipy_dummy
Here is the actual interface with some example usage in the main from other scripts, web interfaces etc.
https://github.com/Garyfallidis/dipy/blob/interfaces/dipy/workflows/dummy.py

Let me know what you think.